### PR TITLE
Add install yarn guidance

### PR DESCRIPTION
**Pull request by bokket**
fix: #84 
I think it may be helpful for other contributors......

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 This website is built using [Docusaurus 2](https://v2.docusaurus.io/), a modern static website generator.
 
+## Install yarn
+- For Debian-derived distros (including Ubuntu and Kali), follow [these](https://yarn.bootcss.com/docs/install/#debian-stable) instructions.
```
Comment (Xuanwo) [inline README.md:6]: Please don't introduce Chinese docs for our English site. And we don't need to specify the OS / Distros here.

Maybe we can refer to https://yarnpkg.com/getting-started/install

We can move this section into `Installation`.

Comment (bokket) [reply README.md:6]: Reasonable

**Review by Xuanwo — APPROVED** [review]

**Comment by bokket** [comment]
Sorry for my initial carelessness , did not put https://yarn.bootcss.com/docs/install/#debian-stable
replace with https://yarnpkg.com/getting-started/install
cry(:

**Comment by Xuanwo** [comment]
Doesn't matter, I found this typo after merged, so I fixed it by myself.

**Pull request merged**